desafio1.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+<!----------------------------------MENU TOPO--------------------------------------------->    
+    <head>
+        <meta charset="utf-8">
+        <header>
+            <figure>
+                <img src="" alt="ImagemBorboletaWomakersCode"/>
+            </figure>
+            <nav class="menu-topo">
+                <ul>
+                    <li> Home </li>
+                    <li> Serviços </li>
+                    <li> Portfólio </li>
+                    <li> Sobre mim </li>
+                    <li> Contato </li>
+                </ul>
+            </nav>
+</header>
+<!-----------------------------------FIM MENU TOPO-------------------------------------------------->    
+    </head>
+
+    <body>
+    <title>Bem Vind@</title>
+    <h2>Curabitur Iaculis</h2>
+    <h3>texto 3 linhas</h3>
+    <!-------------------------------------MENU PRINCIPAL----------------------------------------->  
+    <br>
+    <nav class="menu-principal">
+        <h1> <a href="#">Serviços</a></h1>
+        <table>
+            <tr> 
+                <figure>
+                    <img src="" alt="websites">
+                </figure>
+                <h2><a href="#">Websites</a></h2>
+                <h3>texto</h3>
+            </tr>
+            <tr>
+                <figure>
+                    <img src="" alt="templates">
+                </figure>
+                <h2><a href="#">Templates</a></h2>
```

Comment:
```suggestion
               <a href="#">Templates</a>
```

Não é necessário incluir uma tag de hipertexto e link juntos. Neste caso você pode manter a tag `<a>` depois manipular via CSS o tamanho dela ( usando a propriedade `font-size`).